onisite/settings_local_example.py

```diff
@@ -176,31 +209,6 @@
         'django.middleware.clickjacking.XFrameOptionsMiddleware',
     )
 
-    if os.getenv('ONI_LOG_SQL', 0) == '1':
-        LOGGING = {
-            'version': 1,
-            'disable_existing_loggers': False,
-            'filters': {
-                'require_debug_true': {
-                    '()': 'django.utils.log.RequireDebugTrue',
-                }
-            },
-            'handlers': {
-                'console': {
-                    'level': 'DEBUG',
-                    'filters': ['require_debug_true'],
-                    'class': 'logging.StreamHandler',
-                }
-            },
-            'loggers': {
```

Comment:
out of curiosity, why did you remove this in the new version?

Reply:
It's actually basically been moved.  I combined all registered handlers and filters into the main `LOGGING` definition, so that we didn't have to futz around with large blocks inside the `if` blocks.  The prior approach was a bit unruly to read, but also was really annoying because it was a "clobber the old definition" system.  e.g., if you wanted SQL to be logged, you couldn't have the file-based logging at the same time.

Reply:
Oh I see, I had noticed `'loggers': {},` first, but I see you're adding it back in later with a different handler.  NM